cmd/agentbasedinstaller/register.go

```diff
@@ -62,6 +62,11 @@ func RegisterCluster(ctx context.Context, log *log.Logger, bmInventory *client.A
 	log.Infof("releaseImage version %s cpuarch %s", releaseImageVersion, releaseImageCPUArch)
 
 	clusterParams := controllers.CreateClusterParams(&cd, &aci, pullSecret, releaseImageVersion, releaseImageCPUArch, nil)
+
+	if aci.Spec.Networking.NetworkType != "" {
+		clusterParams.NetworkType = &aci.Spec.Networking.NetworkType
+	}
```

Comment:
In ZTP a default network type is chosen by the controller if none is specified: https://github.com/openshift/assisted-service/blob/release-4.11/internal/controller/controllers/clusterdeployments_controller.go#L908-L937

In master this probably doesn't matter since the default is OVNK for everything, but if we were to backport we'd need the part that explicitly specifies OVNK if IPv6 is enabled.

It might be better to just call `updateNetworkParams()` from inside `CreateClusterParams()` - this is the only field that doesn't get set on create, and that's probably a bug in ZTP (that could potentially cause races).